invenio_circulation/transitions/base.py

```diff
@@ -55,6 +55,10 @@ def inner(self, loan, **kwargs):
             msg = 'Required input parameters are missing `[{}]`'\
                 .format(missing)
             raise TransitionConstraintsViolation(msg=msg)
+        if all(param not in kwargs for param in self.PARTIAL_REQUIRED_PARAMS):
```

Comment:
I would approach this a little bit differently.
You can avoid casting it to a `set` by defining a set (which it is) as `PARTIAL_REQUIRED_PARAMS`
```
if set(self.PARTIAL_REQUIRED_PARAMS).issubset(list(kwargs)):
    # do your stuff

```

Reply:
Ok I will do it with the intersection of the 2 sets.

Reply:
I am thinking that converting both lists and find intersection is a bit overkill just to find if both values are missing. WDYT?